examples/example_cartesian_path.py

```diff
@@ -81,14 +82,21 @@
 plt.title("Joint Position Trajectories")
 for idx in range(q_vec.shape[0]):
     plt.plot(t_vec, q_vec[idx, :])
+
+line = plt.axvline(x=0, color="b")
 plt.legend(model.names[1:])
 plt.show()
 
+curr_time = 0
+
 if success:
     print("Cartesian planning successful!")
     input("Press 'Enter' to animate the path.")
     for idx in range(q_vec.shape[1]):
         viz.display(q_vec[:, idx])
-        time.sleep(dt)
+        plt.pause(dt)
+        curr_time += dt
+        line.remove()
+        line = plt.axvline(x=curr_time, color="b")
```

Comment:
Instead of removing and re-adding the line, you can actually do this, which is more efficient:

```suggestion
        line.set_xdata([curr_time, curr_time])
````